coveralls/api.py

```diff
@@ -21,7 +21,7 @@ class CoverallsException(Exception):
 
 class Coveralls(object):
     config_filename = '.coveralls.yml'
-    api_endpoint = 'https://coveralls.io/api/v1/jobs'
+    default_coveralls_host = 'https://coveralls.io'
```

Comment:
Please allow overriding this variable in `.coveralls.yml` as well as through the environment variable.